test/controllers/tasks_controller_test.rb

```diff
@@ -89,6 +89,20 @@ def in_taxonomy_scope(organization, location = nil)
                      session: set_session_user(User.current)
           assert_response :not_found
         end
+
+        it 'shows duration column in exported task' do
+          task = ForemanTasks::Task.with_duration.find(FactoryBot.create(:some_task).id)
+          get(:show, params: { id: task.id, duration: task.duration }, session: set_session_user)
+          assert_response :success
+        end
+      end
+
+      describe 'index' do
+        it 'shows duration column in exported task' do
+          task = ForemanTasks::Task.with_duration.find(FactoryBot.create(:some_task).id)
+          get(:index, params: { id: task.id, duration: task.duration }, session: set_session_user)
+          assert_response :success
```

Comment:
This test currently tests that a request succeeds, but not that correct data is returned.